### ISSUE TITLE
Move "Databases/Configs" to sqlite

### ISSUE THREAD

**Issue by jztech101** [TechCavern/WaveTact#73]
Json will not work for future endeavors. And this will possibly allow for per channel commands

- http://javalite.github.io/activejdbc/ (Move to using SQLite - https://bitbucket.org/xerial/sqlite-jdbc)
- Merge SimpleAction and SimpleMessage with a boolean
- Add Commands to modify configs at runtime
- Remove Extraneous Registry Entries
- Rework Console Commands

**Comment by jztech101** [comment]
Loading intellij errors:
Welcome to WaveTact!

Extracting natives...
Processing: libjunixsocket-linux-1.5-amd64.so
-> Skipping. File exists.
Processing: libjunixsocket-linux-1.5-i386.so
-> Skipping. File exists.
Processing: libjunixsocket-macosx-1.5-i386.dylib
-> Skipping. File exists.
Processing: libjunixsocket-macosx-1.5-x86_64.dylib
-> Skipping. File exists.
Patching java.library.path...
[2015/01/01 19:24:58] [main] [INFO] org.flywaydb.core.internal.dbsupport.DbSupportFactory - Database: jdbc:sqlite:./db.sqlite (SQLite 3.0)
[2015/01/01 19:24:58] [main] [INFO] org.flywaydb.core.internal.command.DbValidate - Validated 1 migration (execution time 00:00.072s)
[2015/01/01 19:24:58] [main] [INFO] org.flywaydb.core.internal.command.DbMigrate - Current version of schema "main": 1
[2015/01/01 19:24:58] [main] [INFO] org.flywaydb.core.internal.command.DbMigrate - Schema "main" is up to date. No migration necessary.
Getting connection...
Creating DSLContext...
[2015/01/01 19:24:59] [main] [INFO] org.reflections.Reflections - Reflections took 208 ms to scan 1 urls, producing 22 keys and 193 values 
Exception in thread "main" java.lang.ExceptionInInitializerError
	at com.techcavern.wavetact.utils.LoadUtils.initiateDatabaseConnection(LoadUtils.java:33)
	at com.techcavern.wavetact.Main.main(Main.java:27)
Caused by: java.lang.NullPointerException
	at com.techcavern.wavetact.utils.DatabaseUtils.getConfig(DatabaseUtils.java:43)
	at com.techcavern.wavetact.utils.Registry.<clinit>(Registry.java:29)
	... 2 more

Loading a compiled jar errors:
jztech101@home:~/Downloads$ ~/software/jdk/jre/bin/java -jar WaveTact.jar 

Welcome to WaveTact!

Extracting natives...
Processing: libjunixsocket-linux-1.5-amd64.so
Processing: libjunixsocket-linux-1.5-i386.so
Processing: libjunixsocket-macosx-1.5-i386.dylib
Processing: libjunixsocket-macosx-1.5-x86_64.dylib
Patching java.library.path...
Exception in thread "main" org.flywaydb.core.api.FlywayException: Unable to instantiate JDBC driver: org.sqlite.JDBC
	at org.flywaydb.core.internal.util.jdbc.DriverDataSource.<init>(DriverDataSource.java:115)
	at org.flywaydb.core.Flyway.setDataSource(Flyway.java:734)
	at com.techcavern.wavetact.utils.LoadUtils.initiateDatabaseConnection(LoadUtils.java:26)
	at com.techcavern.wavetact.Main.main(Main.java:27)
Caused by: java.lang.ClassNotFoundException: org.sqlite.JDBC
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:340)
	at org.flywaydb.core.internal.util.ClassUtils.instantiate(ClassUtils.java:49)
	at org.flywaydb.core.internal.util.jdbc.DriverDataSource.<init>(DriverDataSource.java:111)
	... 3 more

/me shakes his fist

**Comment by rewbycraft** [comment]
See pull request #88.
The last error is my fault (I derped with the build.gradle).
The last one is on you.
Here's what was going wrong:
When you start the db-connection, you set Registry.WaveTactDB to the DSLContext.
There's nothing wrong with that in and of itself, but you also have lines in there that set other properties to stuff from the database.
When you try to set WaveTactDB java will first initiate the class and try to resolve the other properties.
This will nullpointer because 4 of these properties are trying to fetch something from the db, which hasn't been set yet at this point.
So I deferred those to a separate function which gets called *after* the db has been setup properly.
I also fixed another NPE-waiting-to-happen in getConfig.
